packages/catalog-process/src/repositories/db.ts

```diff
@@ -27,7 +27,7 @@ export const dbConfig: IConnectionParameters<IClient> = {
   host: conData.hostname,
   password: conData.password,
   port: conData.port,
-  ssl: false,
+  ssl: true,
```

Comment:
we probably need an SSL configuration object...

Reply:
Maybe but I want to try with a simple boolean config.
Right now my db client is configured like this
![image](https://github.com/pagopa/interop-be-monorepo/assets/32327779/7919b179-6083-4515-8428-f9a5177675f4)
and these are the possible values of this field
![image](https://github.com/pagopa/interop-be-monorepo/assets/32327779/fdd32675-0f91-4e09-a90d-5f76643e2afd)

In theory, we could use the default value and should be fine